app/main.py

```diff
@@ -1,8 +1,50 @@
+from typing import Any
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int | float,
+                 clean_mark: int | float,
+                 brand: int | float) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
+
+
+car = Car(comfort_class=0, clean_mark=0, brand=0)
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int | float,
+                 clean_power: int | float,
+                 average_rating: int | float,
+                 count_of_ratings: int | float) -> None:
```

Comment:
The type annotations `int | float` are incorrect. Use `Union[int, float]` from the `typing` module for type annotations that allow multiple types.